charts/newrelic-k8s-metrics-adapter/templates/deployment.yaml

```diff
@@ -61,6 +61,10 @@ spec:
             secretKeyRef:
               name: {{ template "newrelic-k8s-metrics-adapter.fullname" . }}
               key: personalAPIKey
+        {{- if $.Values.proxy }}
+        - name: HTTPS_PROXY
```

Comment:
@kang-makes should the pipeline fail asking for a version bump?

Reply:
ahh the index is empty and there is no release due to the github outages, so maybe that is why it is not failing

Reply:
Can you rebase?

Reply:
Done!

Reply:
the pipeline is still not failing 🤔

Reply:
https://github.com/helm/chart-testing/blob/main/doc/ct_lint.md
```
--all  Process all charts except those explicitly excluded.
       Disables changed charts detection and version increment checking
```

We have to fix it [in this repository](https://github.com/newrelic/newrelic-k8s-metrics-adapter/blob/main/.github/workflows/ci.yaml#L68) and [in `nri-kubernetes` one](https://github.com/newrelic/nri-kubernetes/blob/main/.github/workflows/push_pr.yaml#L47).